### PR TITLE
Add OpenSSL to msvc build

### PR DESCRIPTION
**Pull request by anosach-luxoft**
- Change cmake files to include openssl headers
- Fix encryption library build

OpenSSL library should be installed and environment variable OPENSSL_DIR point to
OpenSSL directory

### CODE REVIEW

**Comment by anosach-luxoft** [comment]
@nk0leg, @AGaliuzov, @pestOO, please review

File: src/3rd_party-static/encryption/src/Base64.cpp [hunk 1]
```diff
@@ -82,7 +82,7 @@ std::string base64_encode(unsigned char const* bytes_to_encode, unsigned int in_
 }
 
 std::string base64_decode(std::string const& encoded_string) {
-  int in_len = encoded_string.size();
+  int in_len = (int)encoded_string.size();
```
Comment (pestOO) [inline src/3rd_party-static/encryption/src/Base64.cpp]: May be update in_len type?

File: src/3rd_party-static/encryption/src/Base64.cpp [hunk 2]
```diff
@@ -93,7 +93,7 @@ std::string base64_decode(std::string const& encoded_string) {
     char_array_4[i++] = encoded_string[in_]; in_++;
     if (i ==4) {
       for (i = 0; i <4; i++)
-        char_array_4[i] = base64_chars.find(char_array_4[i]);
+        char_array_4[i] = (unsigned char)base64_chars.find(char_array_4[i]);
```
Comment (pestOO) [inline src/3rd_party-static/encryption/src/Base64.cpp]: May be use static_cast according to gstyle?

**Comment by pestOO** [comment]
Reviewed

File: src/3rd_party-static/encryption/src/Base64.cpp (hunk 1 above)
Comment (AGaliuzov) [inline src/3rd_party-static/encryption/src/Base64.cpp]: Actualt string size is sizet not int

Comment (anosach-luxoft) [inline src/3rd_party-static/encryption/src/Base64.cpp]: I don't think I have to change even variables type in 3rd-party library. I'm going to use static_cast here

**Comment by nk0leg** [comment]
Reviewed

**Pull request merged**